optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/admin/AdminPage.java

```diff
@@ -19,11 +20,19 @@
     @Inject
     @DataField("reset-application-button")
     HTMLButtonElement resetApplicationButton;
+    
+    @Inject
+    private LoadingSpinner loadingSpinner;
+    
+    @Inject
+    private NotificationSystem notificationSystem;
 
     @EventHandler("reset-application-button")
     private void resetApplication(@ForEvent("click") MouseEvent e) {
+        loadingSpinner.showFor("reset-application");
         AdminRestServiceBuilder.resetApplication(null, FailureShownRestCallback.onSuccess((success) -> {
-            ErrorPopup.show("Application was successfully reset");
+            loadingSpinner.hideFor("reset-application");
+            notificationSystem.notify("Application was reset successfully", "Application was reset successfully, please refresh the page.");
```

Comment:
@Christopher-Chianelli  @ge0ffrey Looking at the message I just want to be sure - we don't have to provide i18n internationalization for Employee Rostering, right?

Reply:
@rsynek Not yet. That will be a big refactor indeed that we'll need to do sometime in the future.

Reply:
@ge0ffrey do we have any epic for the refactor? If so, we could start putting "do not forget" issues there.